docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -158,6 +220,25 @@ if the Okta integration is re-enrolled.
 To permanently delete the SAML connector, navigate to the `Auth Connectors`
 page in the Teleport UI and delete it from there.
 
+### Access Lists
+
+<Admonition type="warning">
+If the hosted integration is still active, removing Okta sourced Access Lists
+could revoke Okta access from users in your organization. Please exercise caution
```

Comment:
Feels like the UI should help prevent this. Does it give you a warning?

Reply:
no warning, but that's good feedback, i'll add it when i add okta badge on the access list listing